report_thesis/src/sections/proposed_approach/optimization_framework.tex

```diff
@@ -42,152 +40,125 @@ \subsubsection{The Framework}
 We believe that this approach will allow us to best identify the top-$n$ pipelines to be used in our stacking ensemble.
 
-To manage the optimization process the function receives the number of trials to run, a list of models, and a list of oxides, seen in line~\ref{step:initialize_run_process}, and initializes the optimization environment by setting up the sampler, pruner, and the objective function, seen in lines~\ref{step:initialize_sampler} to~\ref{step:initialize_objective}.
+To manage the optimization process the function receives the number of trials to run, a list of models, and a list of oxides, seen in line~\ref{step:initialize_run_process}, and initializes the sampler, seen in lines~\ref{step:initialize_sampler}.
 
 The sampler is responsible for managing the search space of the hyperparameters for the optimization process.
 This means that any hyperparameters being evaluated, for any preprocessor or model, will be managed by this sampler, which allows us to optimize for all hyperparameters at the same time.
 Optuna provides several options for samplers that have different characteristics and each have their strengths and weaknesses.
 However, because we require multi-objective optimization, this naturally limits the choice of sampler to those that support this.
-For our framework we chose to use the Tree-structured Parzen Estimator (TPE) sampler, due to its stated optimization efficiency and its ability to handle all use cases. 
-Additionally, the TPE sampler allows us to control how many of the trials to be reserved for exploration, which is beneficial when the search space is large. \cite{optuna_2019}
+For our framework we chose to use the \gls{tpe} sampler, due to its stated optimization efficiency and its ability to handle all use cases. 
+Additionally, the TPE sampler allows us to control how many of the trials to be reserved for exploration, which is beneficial when the search space is large\cite{optuna_2019}.
 
-The pruner is responsible for stopping unpromising trials early, to reduce the computational costs of the optimization process and is initialized in line~\ref{step:initialize_pruner}.
-Similar to the sampler, there exist several options for the choice of pruner.
-In our case we chose to use the Hyperband pruner, due to its balanced approach when pruning.
-Initially, the pruner will consider more configurations, but allow them to run for a shorter time.
-Through several iterations, it will gradually reduce the number of configurations being considered, but allow them to run for a longer time.
-This provides a higher likelihood that promising configurations are not pruned too early, while still reducing the computational costs.\cite{optuna_2019}
+Guiding the optimization process is the \nameref{alg:combined_objective} function, which evaluates the performance of each trial, seeking to either minimize or maximize some specified metrics.
```

Comment:
It doesn't seek to do anything but measure performance. The optimizer does the seeking.